mloop/visualizations.py

```diff
@@ -197,30 +198,64 @@ def plot_cost_vs_run(self):
         for ut in self.unique_types:
             artists.append(plt.Line2D((0,1),(0,0), color=_color_from_controller_name(ut), marker='o', linestyle=''))
         plt.legend(artists,self.unique_types,loc=legend_loc)
+    
+    def _ensure_parameter_subset_valid(self, parameter_subset):
+        # Make sure that parameter_subset is iterable.
+        if not hasattr(parameter_subset, '__iter__'):
```

Comment:
Yeah I was mainly adding this for more inexperienced Python users who may try `parameter_subset=1` instead of `parameter_subset=[1]` and might appreciate a very specific error message. I'd be perfectly fine with removing that though.

Reply:
Hm, that's a good point. I think I'd still lean towards removing it though.

Reply:
Sounds good! Removed in 6ec5794.